packages/hash/frontend/src/pages/[account-slug]/types/entity-type/[entity-type-id].page.tsx

```diff
@@ -132,6 +139,41 @@ const Page: NextPageWithLayout = () => {
 
   const entityType = remoteEntityType ?? draftEntityType;
 
+  useEffect(() => {
+    if (authenticatedUser && !namespace) {
+      void router.replace(
+        `/@${authenticatedUser.shortname}/types/new/entity-type`,
+      );
+    }
+  }, [authenticatedUser, namespace, router]);
+
+  useEffect(() => {
+    if (authenticatedUser && !namespace) {
+      // eslint-disable-next-line no-console
+      console.error(
+        `Error: Couldn't find namespace with shortname '${router.query["account-slug"]}'.`,
+      );
+      void router.replace(
+        `/@${authenticatedUser.shortname}/types/new/entity-type`,
+      );
+      return;
+    }
```

Comment:
Curious why this is here twice?

Reply:
Hmm I think I might have overlooked this when resolving merge conflicts. Removed the duplicated code in a1c372beb8599bda17749144e182f34a8577ea6f.